README.md

```diff
@@ -5,24 +5,44 @@ This is an android application that is being created to gain experience developi
 applications written in Java, using git, solidifying programming fundamentals, and designing
 an application from markup to deployment.
 
+#### Code Rules
+  * SV Go shall be written in Java. 
+  * Application shall be written with strong variable typing.  
+     * This is to build good coding practices, and to have code written as implicit as possible. 
+  * Each line shall have a 100 character limit.  
+     * The is to improve code readability as it is
+   easier to read code in a digestible format.
+  * Code shall be properly formatted.  
+    * This is to maintain consistency throughout all of the code files.
+  * Every feature shall introduced will be thoroughly testing using unit testing as well as instrumented 
+  testing.  
+     * This is to improve the UX.
```

Comment:
And finally, mention that we are developing with TDD (Test-Driven Development) in mind.  That is, we write tests before we write code so we are thinking about how the code should be written, testing both the happy path(s) and failure states and their intended outcomes, etc